homework-1-Arrays/arrays.js

```diff
@@ -0,0 +1,44 @@
+let initialValues = [12, "7", undefined, 5, true, "finally", [1, 2, 3], false];
```

Comment:
in loc de let mergea un const. Se aplica peste tot unde ai let.
let se pune doar daca acea variabila isi va schimba, in timp, tipul de data sau valoarea pe care o retine. Cum la tine, in exemplul acesta, tipul de data este mereu un array, ea poate fi mereu constanta de tip array.